crypto/etls/encrypt.go

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright 2018 The ThunderDB Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package etls
+
+import (
+	"crypto/aes"
+	"crypto/cipher"
+	"crypto/rand"
+	"io"
+
+	ec "github.com/btcsuite/btcd/btcec"
+	"github.com/thunderdb/ThunderDB/crypto/hash"
+)
+
+// KeyDerivation, according to ANSI X9.63 we should do a key derivation before using
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
comment on exported function KeyDerivation should be of the form "KeyDerivation ..."